src/dao/AbstractContractDAO.js

```diff
@@ -394,29 +400,36 @@ export default class AbstractContractDAO extends EventEmitter {
     args: Array = [],
     infoArgs: Object | AbstractModel = null,
     value: BigNumber = new BigNumber(0),
-    addDryRunFrom = null,
-    addDryRunOkCodes = [],
-    allowNoReturn = false
+    options = DEFAULT_TX_OPTIONS
```

Comment:
_multisigTx broken. Review and test all _txs and _multisigTx signatures.

Reply:
Fixed